src/main/java/formflow/library/pdf/DataBaseFieldPreparer.java

```diff
@@ -24,14 +28,20 @@ public List<SubmissionField> prepareSubmissionFields(Submission submission) {
 
     dbFields.forEach((fieldName, value) -> {
       switch (fieldName) {
-        case "submittedAt" -> databaseFields.add(new DatabaseField(fieldName, submission.getSubmittedAt().toString()));
+        case "submittedAt" -> databaseFields.add(new DatabaseField(fieldName, formatDateTime(submission.getSubmittedAt())));
         case "submissionId" -> databaseFields.add(new DatabaseField(fieldName, submission.getId().toString()));
-        case "createdAt" -> databaseFields.add(new DatabaseField(fieldName, submission.getCreatedAt().toString()));
-        case "updatedAt" -> databaseFields.add(new DatabaseField(fieldName, submission.getUpdatedAt().toString()));
+        case "createdAt" -> databaseFields.add(new DatabaseField(fieldName, formatDateTime(submission.getCreatedAt())));
+        case "updatedAt" -> databaseFields.add(new DatabaseField(fieldName, formatDateTime(submission.getUpdatedAt())));
         case "flow" -> databaseFields.add(new DatabaseField(fieldName, submission.getFlow()));
         default -> log.error("Unable to map unknown database field: {}", fieldName);
       }
     });
     return databaseFields;
   }
+
+  private String formatDateTime(Date date) {
+    DateTimeFormatter formatter = DateTimeFormatter.ofLocalizedDate(FormatStyle.SHORT).withLocale(Locale.US).withZone(
```

Comment:
This all looks good. One question this raises for me is the locale part. We might need to configure the system for a default timezone to work in that's separate from the timezone the system is running in.

Reply:
Could you please make the year 4 digits?  The DateTimeFormatter allows for patterns like this:  
`DateTimeFormatter formatter = DateTimeFormatter.ofPattern("MM/dd/yyyy");`

https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/time/format/DateTimeFormatter.html